microprofile.jdt/org.eclipse.lsp4mp.jdt.test/src/main/java/org/eclipse/lsp4mp/jdt/core/config/MicroProfileConfigJavaDiagnosticsTest.java

```diff
@@ -0,0 +1,54 @@
+package org.eclipse.lsp4mp.jdt.core.config;
+
+import static org.eclipse.lsp4mp.jdt.core.MicroProfileForJavaAssert.assertJavaDiagnostics;
+import static org.eclipse.lsp4mp.jdt.core.MicroProfileForJavaAssert.d;
+
+import java.util.Arrays;
+
+import org.eclipse.core.resources.IFile;
+import org.eclipse.core.runtime.Path;
+import org.eclipse.jdt.core.IJavaProject;
+import org.eclipse.lsp4j.Diagnostic;
+import org.eclipse.lsp4j.DiagnosticSeverity;
+import org.eclipse.lsp4mp.commons.DocumentFormat;
+import org.eclipse.lsp4mp.commons.MicroProfileJavaDiagnosticsParams;
+import org.eclipse.lsp4mp.jdt.core.BasePropertiesManagerTest;
+import org.eclipse.lsp4mp.jdt.core.utils.IJDTUtils;
+import org.eclipse.lsp4mp.jdt.internal.faulttolerance.MicroProfileFaultToleranceConstants;
+import org.eclipse.lsp4mp.jdt.internal.faulttolerance.java.MicroProfileFaultToleranceErrorCode;
+import org.eclipse.lsp4mp.jdt.internal.metrics.MicroProfileMetricsConstants;
+import org.junit.Test;
+
+/**
+ * MicroProfile Fault Tolerance definition in Java file.
+ *
+ * @author Angelo ZERR
```

Comment:
Update the author

Reply:
I've just kept it empty without author reference. Let me know if you prefer I list myself or anything like that. I've seen some files with / some without.